string FileName
string DataPath
int8 Lap
RoiList[] Rois
bool TestVideo

